Mh version beta 0.0.4

bleehu opened the pull request:
This should be put on hold until the version beta 0.0.3 has been reviewed and merged.


PR #305  first.

bleehu committed:
added armor peircing levels to medic attacks

bleehu committed:
Balanced Armor piercing levels for medic proceedures. Also spell checked.

bleehu committed:
Changed to-miss percentages to conform with Combat 2.0 standards

bleehu committed:
started work removing nanites from engineering

bleehu committed:
added constituent elements of engineer processes

bleehu committed:
added engineer's consumable items and put limits on engineer's elements.

bleehu committed:
added level progression limits and removed nanite gen/cost

bleehu committed:
transitioned several engineer processes to use energy and material instead of nanites

bleehu committed:
Rounded half accuracy modifiers

bleehu committed:
Merge branch 'dev' into MH-update-Engineer

 Conflicts:
	CharacterCreation/02_Classes.txt

 Changes to be committed:
	modified:   ../Basic Rules.txt
	modified:   02_Classes.txt
	modified:   03_Feats.txt
	renamed:    04_02_WeaponAttachments.txt -> 04_00_WeaponAttachments.txt
	new file:   04_01_Melee_&_Thrown_Weapons.txt
	deleted:    04_01_Weapons.txt
	new file:   04_02_Pistols.txt
	deleted:    04_03_Advanced Weapons.txt
	new file:   04_03_SMGs_&_Shotguns.txt
	new file:   04_04_Carbines_AssaultRifles_&_BattleRifles.txt
	new file:   04_05_Long_Rifles_&_DMRs.txt
	new file:   04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt
	modified:   05_ArmourList.txt
	modified:   06_Skills.txt
	deleted:    Character Sheets/Daeron_CharSheet.txt
	deleted:    Character Sheets/Monk_Hirito_12.txt
	deleted:    Character Sheets/Paladin_Tibor_6.txt
	deleted:    Character Sheets/[TEMPLATE](CHARNAME)_CharSheet.txt
	modified:   Class-Specific Documentation/Medic Proceedures.txt
	modified:   Class-Specific Documentation/Medic Skill Tree - Malpractice.png
	modified:   Class-Specific Documentation/Tome of Spells.txt
	new file:   ../Glossary_of_Terms.txt

bleehu committed:
change races to have the agreed upon style of headers

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

bleehu committed:
Stylized Races and added some headings

bleehu committed:
Changed backtick character to ascii-compliant apostrophe

bleehu committed:
one more backtick

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

 Conflicts:
	CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

bleehu committed:
added flavor note to free running.

bleehu committed:
Removed unused Nanite Push Proceedure

bleehu committed:
grammar check, scaled Harmacist spike and fixed shield penetration language

bleehu committed:
stylized Medic Proceedures, added payload clarification and fixed some language on defeating armor

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

bleehu committed:
spell procedures like the dictionary does

bleehu committed:
updated intro paragraph

bleehu committed:
Medic isn't a class. Renamed Document

bleehu committed:
It's not called Medic, it's called doctor. Also Changed "Quarentine" to "Contaminate Area".
Added some style changes.
 Changes to be committed:
	renamed:    Medic Skill Tree - Hippocratic.png -> Doctor Procedure Tree - Hippocratic.png
	new file:   Doctor Procedure Tree - Malpractice.png
	modified:   Doctor Procedures.txt
	deleted:    Medic Skill Tree - Malpractice.png

bleehu committed:
finished stylizing the Proceedures Doc

bleehu committed:
added whateverectomy

1sourcecontrol committed:
Update 05_ArmourList.txt

1sourcecontrol committed:
Update 05_ArmourList.txt

missed a spot

1sourcecontrol committed:
update APs to AS and grammer/styleguide

bleehu committed:
Spell check and copywrite removal.

bleehu committed:
added space for consistency

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

also cleaned up racial traits

 Conflicts:
	CharacterCreation/01_Races.txt

bleehu committed:
Stylized Basic Rules

bleehu committed:
Added Free Action to Glossary

bleehu committed:
some small typos

bleehu committed:
updated buzzed to conform to Combat 2.0

bleehu committed:
Merge pull request #252 from bleehu/MH-Issue-251-Buzzed2.0

updated buzzed to conform to Combat 2.0

1sourcecontrol committed:
Merge pull request #248 from bleehu/address#198_nStyleGuide_AL

merge to dev,  for issue #198. Styleguide

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

bleehu committed:
Some simple cleanup

bleehu committed:
Stylized Items document.

bleehu committed:
Merge pull request #255 from bleehu/MH-version-beta-0.0.2-bump

bumped version number

bleehu committed:
Some grammar and capitolization details cleaned up

bleehu committed:
Lowered prices on Anfo and breaching charges

bleehu committed:
removed unnecessary section header from engineering processes

bleehu committed:
Merge pull request #257 from bleehu/MH-Stylize-Engineer-Processes

Stylize Engineer Process Docs

bleehu committed:
removed old pre-beta sheet

bleehu committed:
Stylized Basic Rules to conform with proposed table styles

bleehu committed:
Merge pull request #260 from bleehu/MH-remove-old-sheet

removed old pre-beta sheet

bleehu committed:
added glow sticks to items

Turtlelord26 committed:
Language Deletion Pending Update

re #259

bleehu committed:
Merge pull request #202 from bleehu/MH-medic-2.0

Mh medic 2.0

Turtlelord26 committed:
Merge pull request #266 from bleehu/Turtlelord26-Temp_Languages_Removal

Language Deletion Pending Update

bleehu committed:
added glowsticks but for real

bleehu committed:
Merge pull request #261 from bleehu/MH-Basic-Table-Styles

Stylized Basic Rules to conform with proposed table styles

bleehu committed:
removed copywrite compound from glowsticks

just to be safe.

bleehu committed:
Update repair kit to match med kit

why use two paragraphs when one will do?

bleehu committed:
Merge pull request #263 from bleehu/MH-add-Glowsticks

Added glow sticks to items

Turtlelord26 committed:
Stylize New Player Walkthrough

re Issue #236

Turtlelord26 committed:
Merge pull request #267 from bleehu/Turtlelord26-Stylize_Walkthrough

Stylize New Player Walkthrough

Turtlelord26 committed:
Skills doc draft from hulk

Turtlelord26 committed:
Add files via upload

Turtlelord26 committed:
Readability enabled

Turtlelord26 committed:
Rename Skills Draft.txt to 06_01_Systemized_Skills.txt

Turtlelord26 committed:
Merge pull request #268 from bleehu/Turtlelord26-skillsdraft

Skills doc draft from hulk

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

Turtlelord26 committed:
Update 07_Items.txt

Turtlelord26 committed:
Addressing Racial Stylization Review

Handled all blocking and nonblocking issues from my earlier review, except for the inconsistent handling of racial trait style. I didn't want to unilaterally pick a way to do that, as I'm not on the team responsible for generating the traits.

bleehu committed:
took out the trash in races doc

Turtlelord26 committed:
Merge pull request #282 from bleehu/Turtlelord26-Stylize-Races_Review

Addressing Racial Stylization Review

bleehu committed:
Merge pull request #226 from bleehu/MH-Stylize-Races

Stylized Races document

Turtlelord26 committed:
Addressing Basic Rules Stylization Review

Fixed all blockers and nonblockers from my recent review, except tables.

Turtlelord26 committed:
Merge pull request #281 from bleehu/Turtlelord26-Stylize-Items_Review

Addressing Items Stylization Review

bleehu committed:
ordered top directory and added conditions file

bleehu committed:
added and styled conditions

bleehu committed:
Merge branch 'dev' into MH-Stylize-Items

bleehu committed:
Merge pull request #258 from bleehu/MH-Stylize-Items

Stylized Items document

bleehu committed:
added hazmat suits

Turtlelord26 committed:
Partial Reversion

bleehu committed:
Merge pull request #285 from bleehu/MH-Issue-277-Hazmats

added hazmat suits

Turtlelord26 committed:
Merge pull request #284 from bleehu/Turtlelord26-Glossary-Style-Fixes_Review

Addressing Basic Rules Stylization Review

bleehu committed:
Merge branch 'dev' into MH-Glossary-Style-Fixes

branch caught it comittee while other work is being done. Need to not squash that work.

bleehu committed:
Merge pull request #250 from bleehu/MH-Glossary-Style-Fixes

Glossary  and Basic Rules style fixes

bleehu committed:
terminated table with a newline

bleehu committed:
Merge branch 'dev' into MH-conditions-section

Reveraine committed:
Added Roll Definitions to Glossary

Average rolling was something I thought of while reworking races. I don't think it's too complex, but it will need some testing, I think. I'll put more thoughts for it in basic rules.

Reveraine committed:
Rolling the Average to Base Rules

bleehu committed:
removed redundant conditions section

Reveraine committed:
Added reworked Tundari

There are a few things missing. What a weakness to poison looks like and STAT bonuses are not in yet.

Reveraine committed:
Update Glossary_of_Terms.txt

Reveraine committed:
Merge pull request #287 from bleehu/Reveraine-patch-1

Added Roll Definitions to Glossary

Reveraine committed:
Merge pull request #288 from bleehu/Reveraine-patch-2

Rolling the Average to Base Rules

bleehu committed:
Merge branch 'dev' into MH-conditions-section

bleehu committed:
Merge pull request #286 from bleehu/MH-conditions-section

Mh conditions section

Reveraine committed:
Update 01_03_Races_Tundari

Reveraine committed:
Merge pull request #289 from bleehu/Reveraine-Race-Updates

Added reworked Tundari

Turtlelord26 committed:
Stylize Tundari Document

Bringing Tundari document into style compliance.

Of note, I have changed the Clans section header into a Tundari Clans chapter header, in order to make the header hierarchy of that section behave. If we prefer not to separate these into different chapters, we can instead leave Clans as a section and have both the clan and its traits be separate subsections a la 

	===== Clans =====
	
	== Rra’mau ==
	
	== Rra'mau Traits ==

Reveraine committed:
Merge pull request #291 from bleehu/Turtlelord26-Stylize_Tundari

Stylize Tundari Document

bleehu committed:
Merge branch 'dev' into MH-update-Engineer

bleehu committed:
removed many nanite references from Engineer Process Document

bleehu committed:
removed Nanites from Engineer

bleehu committed:
removed one more nanite reference from Engineering Processes

bleehu committed:
removed all of the mentions of nanites from Engineer Processes

Turtlelord26 committed:
Added poison weakness rule.

re issue #290

Turtlelord26 committed:
Ugh typo

Fed it to a starving reptile.

Turtlelord26 committed:
Further typography

Fed to a less-starving reptile.

Turtlelord26 committed:
Merge pull request #293 from bleehu/Turtlelord26-Tundari&Poison

Added poison weakness rule.

Turtlelord26 committed:
Implementing Movement in Microgravity

I decided for the sake of complexity to allow for a lot of GM Discretion for things like angles and collision physics. I've also focused almost entirely on movement, restricting the effects of weapons fire to increased difficulty of Anchoring oneself.

I've also hijacked the 02 slot for nonstandard environmental rules. This is of course subject to change on request.

Remember that this has 'Supplemental Rulebook Material' written all over it.

Turtlelord26 committed:
Review Notes

Turtlelord26 committed:
Merge pull request #294 from bleehu/Turtlelord26-Microgravity_Movement

Implementing Movement in Microgravity

bleehu committed:
added some shield and armor definitions to glossary

bleehu committed:
Revert "added some shield and armor definitions to glossary"

This reverts commit 5b96306495c05b1156f09a95014407bd9c0113b5.

bleehu committed:
Revert "Revert "added some shield and armor definitions to glossary""

This reverts commit 5194e40ff5ff9f8262024452096c8aebf1c76a62.

bleehu committed:
cleaned up grammar of glossary

bleehu committed:
some feat styling

bleehu committed:
Revert "some feat styling"

This reverts commit 84b9efd0308f465ab1da68a97790a6ddcb380c7e.

bleehu committed:
made updates as requested by Engineer review.

bleehu committed:
Merge pull request #301 from bleehu/MH-Issue-300-Armor-Glossary

added some shield and armor definitions to glossary

bleehu committed:
fixes that don't fix things aren't fixes

bleehu committed:
Merge pull request #292 from bleehu/MH-update-Engineer

Mh update engineer

bleehu committed:
bumped version number

bleehu committed:
Merge pull request #304 from bleehu/MH-version-beta-0.0.3

bumped version number

bleehu committed:
stylized heavy weapons

bleehu committed:
Merge pull request #306 from bleehu/MH-Style-Weapons

stylized heavy weapons

bleehu committed:
made style changes to pistols to make prettier after parsing in cxdocs

bleehu committed:
stylized the shotguns

bleehu committed:
fixed slug damage for Mathews Model 3x12 which was accidentally changed in styling. Yanked "slug" from frontier plasma shotgun which doesn't use them.

bleehu committed:
added a couple of blank lines to pistols for ease of reading

bleehu committed:
Merge pull request #308 from bleehu/MH-Stylize-handcannons

made style changes to pistols to make prettier after parsing in cxdocs

bleehu committed:
touched up style of gameplay loop to fix issue #302 and help display of it on cxdocs

bleehu committed:
Merge pull request #309 from bleehu/MH-Style-Weapons

stylized the shotguns

bleehu committed:
Merge pull request #310 from bleehu/MH-Issue-302-style-GL

touched up style of gameplay loop

Turtlelord26 committed:
Manual Merge Artifact Deletion

Turtlelord26 committed:
Miscellaneous Style Touch-ups

Turtlelord26 committed:
Add Formal Aiming

Turtlelord26 committed:
Formal Aiming Typography

Turtlelord26 committed:
Merge pull request #311 from bleehu/Turtlelord26-Formal_Aiming

Turtlelord26 Formal Aiming

1sourcecontrol committed:
Rearrange armor rigs

1sourcecontrol committed:
Modifying Rolling the Average

1sourcecontrol committed:
Delete 04_01_Weapons_L10.txt

1sourcecontrol committed:
Delete 04_02_WeaponAttatchments_L10.txt

1sourcecontrol committed:
Delete 05_Armour_L10.txt

1sourcecontrol committed:
Delete 07_Items_L10.txt

1sourcecontrol committed:
update roll - average

1sourcecontrol committed:
Merge pull request #320 from bleehu/remove-lvl10-folder-AlexL

Remove lvl10 folder

1sourcecontrol committed:
removed roll-the-average strategic advice

1sourcecontrol committed:
Merge pull request #319 from bleehu/Textmod-RollingtheAvg_AlexL

Modifying Rolling the Average

1sourcecontrol committed:
Merge pull request #318 from bleehu/Rearrange-armor-rigs-AlexL

Rearrange armor rigs

1sourcecontrol committed:
defining auto and burst fire rates

also clarified some text that defines "no crits on high fire rate weapons"

1sourcecontrol committed:
removing colloquialism 'crit'

1sourcecontrol committed:
Merge pull request #321 from bleehu/autoburst_fire_rates_AlexL

defining auto and burst fire rates

bleehu committed:
replaced "Silenced" With "Disrupted" to fit aesthetic.

bleehu committed:
Revert "replaced "Silenced" With "Disrupted" to fit aesthetic."

This reverts commit 16c551703e6ca4e65b71fbdb58ae4d407fdee46a.

bleehu committed:
bumped version number to 0.0.4

Turtlelord26 reviewed (commented):
I question 
-sending the new player walkthrough to RC in its currently incomplete state, 
-that hackers are still called priests in classes but referred to as hackers in new skills,
-that we would merge new skills without deleting old skills, 
-and merging the new tundari document before the rest of the race reworks go live.
Other than that, things seem to be in order. Approval awaits only that the above points be spoken to.

bleehu committed:
Remove files that are not ready for R_C in version Beta 0.0.4

 Changes to be committed:
	deleted:    00_New_Player_Walkthrough.txt
	deleted:    01_03_Races_Tundari
	deleted:    06_01_Systemized_Skills.txt

bleehu commented:
Then let's remove what's not ready to be merged to RC. What do you think?

Turtlelord26 reviewed (approved):
:+1:
